code/modules/mob/living/simple_animal/hostile/spaceworms.dm

```diff
@@ -0,0 +1,335 @@
+
+//Worm Segments, Dummy, has no AI, relies on the head.
+/mob/living/simple_animal/hostile/spaceWorm
+	name = "space worm segment"
+	desc = "A part of a space worm."
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "spaceworm"
+	icon_living = "spaceworm"
+	icon_dead = "spacewormdead"
+	status_flags = 0
+
+	speak_emote = list("screeches")
+	emote_hear = list("screeches")
+
+	response_help  = "touches"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+
+	harm_intent_damage = 2
+
+	maxHealth = 50
+	health = 50
+
+	stop_automated_movement = 1
+	animate_movement = SYNC_STEPS
+
+	minbodytemp = 0
+	maxbodytemp = 350
+	min_oxy = 0
+	max_co2 = 0
+	max_tox = 0
+
+	a_intent = "harm" //so they don't get pushed around
+
+	environment_smash = 1
+
+	speed = -1
+
+	AIenabled = 0 //The body isn't conscious
+
+	anchored = 1 //otherwise people can literally fucking pull spaceworms apart
+
+//	faction = list("spaceworms") - Pretty sure this is only a thing on -tg- code
```

Comment:
You do need to set the faction as something, otherwise it'll be neutral and not attack anybody. I think the issue is that it's a list where it shouldn't be.